osu.Game/Screens/Play/BreaksOverlay/GlowIcon.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics;
+using osu.Game.Graphics;
+using OpenTK;
+using osu.Framework.Allocation;
+
+namespace osu.Game.Screens.Play.BreaksOverlay
+{
+    public class GlowIcon : Container
+    {
+        private const int blur_sigma = 5;
+
+        private readonly SpriteIcon spriteIcon;
+        private readonly SpriteIcon glowIcon;
+        private readonly BufferedContainer glowContainer;
+
+        public override Vector2 Size

```

Comment:
don't override size and give it a different meaning.

Reply:
Should i add my own property instead?